shark/iree_utils/compile_utils.py

```diff
@@ -535,13 +535,13 @@ def export_iree_module_to_vmfb(
 ):
     # Compiles the module given specs and saves it as .vmfb file.
     flatbuffer_blob = compile_module_to_flatbuffer(
-        module,
-        device,
-        mlir_dialect,
-        model_config_path,
-        extra_args,
-        debug,
-        compile_str,
+        model=module,
+        device=device,
+        frontend=frontend,
```

Comment:
You can change this to `frontend=mlir_dialect` -- it's not ideal but they are used pretty interchangeably at this point and don't matter much at the moment since we are mostly using PyTorch.

Reply:
(See CPU CI - https://github.com/nod-ai/SHARK/actions/runs/6666541497/job/18118262556?pr=1921#step:8:1614 )